packages/mediacenter/p8-platform/package.mk

```diff
@@ -16,18 +16,19 @@
 #  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
 ################################################################################
 
-PKG_NAME="platform"
-PKG_VERSION="081032f"
+PKG_NAME="p8-platform"
+PKG_VERSION="38343e0"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.kodi.tv"
-PKG_URL="$DISTRO_SRC/$PKG_NAME-$PKG_VERSION.tar.xz"
+PKG_URL="https://github.com/Pulse-Eight/platform/archive/$PKG_VERSION.tar.gz"
+PKG_SOURCE_DIR="platform-$PKG_VERSION*"
```

Comment:
`PKG_SOURCE_DIR="platform-$PKG_VERSION*"` is not required (and `platform-$PKG_VERSION*` is also incorrect).